Formula/rethinkdb.rb

```diff
@@ -26,13 +26,13 @@ class Rethinkdb < Formula
 
   uses_from_macos "curl"
 
-  def install
-    args = ["--prefix=#{prefix}"]
-    args += ["--allow-fetch"] if build.head?
+  patch :DATA
```

Comment:
Please link above this patch to where this has been committed/submitted to the upstream repository.

Reply:
@MikeMcQuaid is this for me of for @jonchang ?

Reply:
Whichever of you added the patch 😁